app/jobs/isolate_job.rb

```diff
@@ -196,4 +201,20 @@ def strip_output(output)
   rescue ArgumentError
     return output
   end
-end
+
+  def call_webhooks(submission)
```

Comment:
Function signature could be just `def call_webhooks` because `submission` is available as class member.

Reply:
Noted, will fix